src/Carbon/Carbon.php

```diff
@@ -1209,6 +1209,25 @@ public static function getWeekendDays()
         return static::$weekendDays;
     }
 
+    /**
+     * Get the all dates of week
+     *
+     * @return array
+     */
+    public static function getCurrentWeekDays()
+    {
+        $startOfWeek = Carbon::instance((new static)->startOfWeek());
```

Comment:
If change this to Carbon::now()->startOfWeek()->subDay() you can then just loop from 0 to < DAYS_PER_WEEK and loop code will stay the same.